src/app/(app)/funds/actions/get-fund-by-id.ts

```diff
@@ -0,0 +1,24 @@
+"use server";
+
+import { Tables } from "@/lib/db";
+import { getSupabaseClient } from "@/utils/supabase/supabaseClient";
+
+export const getFundDetailsById = async (fundId: number) => {
+  const supabase = await getSupabaseClient();
+  const { data, error } = await supabase
```

Comment:
Would it make sense to rename the data variable to something more descriptive?

Reply:
@Mubashar1009 I have improved my interface using the extended method.